src/effects/native/tremoloeffect.cpp

```diff
@@ -51,27 +51,27 @@ EffectManifest TremoloEffect::getManifest() {
     shape->setDefault(0.5);
     shape->setMaximum(0.9);
 
-    EffectManifestParameter* smooth = manifest.addParameter();
-    smooth->setId("smooth");
-    smooth->setName(QObject::tr("Smooth"));
-    smooth->setShortName(QObject::tr("Smooth"));
-    smooth->setDescription(QObject::tr("Sets the smoothness of the modulation"
-    "0 - Square wave"
+    EffectManifestParameter* waveform = manifest.addParameter();
+    waveform->setId("waveform");
+    waveform->setName(QObject::tr("Waveform"));
+    waveform->setShortName(QObject::tr("Waveform"));
+    waveform->setDescription(QObject::tr("Sets the waveform of the modulation\n"
+    "0 - Square wave\n"
```

Comment:
Users can't explicitly set the numeric value of parameters, so use "Fully left" and "Fully right" instead of "0" and "1".